src/calculateRentalCost.js

```diff
@@ -5,6 +5,20 @@
  */
 function calculateRentalCost(days) {
```

Comment:
The function name 'calculateRentalCost' is descriptive and clearly indicates its purpose.